server/src/main/java/org/apache/druid/server/lookup/cache/LookupLoadingSpec.java

```diff
@@ -44,8 +47,8 @@ public enum Mode
     ALL, NONE, ONLY_REQUIRED
   }
 
-  private final Mode mode;
-  private final ImmutableSet<String> lookupsToLoad;
+  private Mode mode;
```

Comment:
These changes should be reverted along with the other changes made to this class.

Reply:
Done, please check the latest changes. Please let me know if this wasn't based on this suggestion: https://github.com/apache/druid/pull/16358#issuecomment-2098326253